mantidproject/mantid
# Enable plotting of individual functions in FitPropertyBrowser

**Description of work.**
This PR allows the user to plot individual functions in the fit property browser. This replicates a feature which was present in MantidPlot. 

**To test:**
- Open workbench
- Load some data
- Plot and open the fit browser
- Add any three functions
- Plot each one by right clicking on the function
- Delete the first function
- The plot should update

Generally, just play around with the fit browser and make sure everything works.

Fixes #29622 . 






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
